test/graphql/mutation/business-create.test.js

```diff
@@ -250,4 +255,112 @@ describe('business', () => {
       }
     })
   })
+
+  test('create a business - withUprn', async () => {
+    delete input.address.withoutUprn
+    delete input.correspondenceAddress.withoutUprn
+
+    const inputWithUprn = {
+      ...input,
+      address: {
+        withUprn: {
+          ...input.address.withoutUprn,
+          uprn: '123456789012'
+        }
+      },
+      correspondenceAddress: {
+        withUprn: { ...input.correspondenceAddress.withoutUprn, uprn: '123456789012' }
+      }
+    }
```

Comment:
There's no need to delete these fields, they get overridden when you build the new input anyway... although then you copy in the fields of the now non-existent "without" objects!? :confounded: 
```suggestion
    const inputWithUprn = {
      ...input,
      address: {
        withUprn: {
          ...input.address.withoutUprn,
          uprn: '123456789012'
        }
      },
      correspondenceAddress: {
        withUprn: { ...input.correspondenceAddress.withoutUprn, uprn: '123456789012' }
      }
    }
```
... or if you don't want the "without" fields, just don't add them:
```suggestion
    const inputWithUprn = {
      ...input,
      address: { withUprn: { uprn: '123456789012' } },
      correspondenceAddress: { withUprn: { uprn: '123456789012' } }
    }
```